src/hooks/useUploadImage.ts

```diff
@@ -0,0 +1,28 @@
+import {Image} from 'react-native-compressor';
+import {composeImageMeta} from '../utils/string/file';
+import {uploadPhoto} from '../service/file';
+
+const useUploadImage = () => {
+  const uploadPhotoImage = async (pathImg: string) => {
+    try {
+      const compressionResult = await Image.compress(pathImg, {
+        compressionMethod: 'auto'
+      });
+      console.log('kaki', compressionResult);
+      const asset = new FormData();
+      asset.append('photo', composeImageMeta(compressionResult));
+
+      const responseUpload = await uploadPhoto(asset);
+      console.log({responseUpload}, 'lahap');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Same as above, avoid using `console.log` in production code. 

```diff
- console.log({responseUpload}, 'lahap');
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->